src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/runtime/JavaThread.java

```diff
@@ -33,12 +33,9 @@
 import sun.jvm.hotspot.utilities.Observable;
 import sun.jvm.hotspot.utilities.Observer;
 
-/** This is an abstract class because there are certain OS- and
-    CPU-specific operations (like the setting and getting of the last
-    Java frame pointer) which need to be factored out. These
-    operations are implemented by, for example,
-    SolarisSPARCJavaThread, and the concrete subclasses are
-    instantiated by the JavaThreadFactory in the Threads class. */
+/** This class is no longer abstract. Platform dependent functionality is now implmented
```

Comment:
```suggestion
/** Platform dependent functionality is now implemented
```
"no longer" very quickly loses context if you don't know when it was abstract, or why.

Reply:
Ok. I can probably just remove the comment. I guess I worded it the way I did to help the reviewer understand (as opposed to a comment just disappearing).